lib/package.ts

```diff
@@ -34,6 +34,18 @@ export class Package {
     return relative(this.pathWorkspace, this.path);
   }
 
+  get workspacePatterns(): string[] {
```

Comment:
I think this logic might now duplicate the logic merged in this PR?

https://github.com/bmish/check-dependency-version-consistency/pull/381/files#diff-35a874d313a6ceef9a49715909ef0b4f276c875a6764bfb738281ed3f986ff4b

Reply:
I think this duplication is okay for now because one version throws exceptions and one doesn't. And one depends on us having a Package instance while the other doesn't.